Wikipedia/mw-bridge/CommunicationBridge.m

```diff
@@ -26,6 +27,7 @@ - (CommunicationBridge*)initWithWebView:(UIWebView*)targetWebView;
 
         __weak CommunicationBridge* weakSelf = self;
         [self addListener:@"DOMContentLoaded" withBlock:^(NSString* type, NSDictionary* payload) {
+            [weakSelf.webView wmf_enableJavascriptToXcodeConsoleLogging];
```

Comment:
agree w/ corey about doing this in debug mode only.

also, why are we introducing a proprietary logging function? let's just replace `console.log` with a native version that reroutes to `DDLog`.

as an aside, i've been thinking we should also add a native `window.onerror` handler so we can (at least in debug mode) throw assertions on JS exceptions so we catch them faster.